sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -65,6 +65,18 @@
         Utils.simpleObjectMapper.registerModule(new AfterburnerModule());
     }
 
+    public static byte[] getUTF8BytesOrNull(String str) {
+        if (str == null) {
+            return null;
+        }
+
+        try {
+            return str.getBytes("UTF-8");
```

Comment:
Instead of using String "UTF-8", we should use the import -> StandardCharsets.UTF_8
don't need to change on this PR as CI have passed, but please change it in next iteration in another PR.

Reply:
you are right. will fix. missed this one.

Reply:
will fix in the next PR. thank you